numpy/_core/src/npysort/highway_qsort.hpp

```diff
@@ -1,8 +1,22 @@
 #ifndef NUMPY_SRC_COMMON_NPYSORT_HWY_SIMD_QSORT_HPP
 #define NUMPY_SRC_COMMON_NPYSORT_HWY_SIMD_QSORT_HPP
 
+#include "hwy/highway.h"
+
 #include "common.hpp"
 
+// This replicates VQSORT_ENABLED from hwy/contrib/sort/shared-inl.h
+// without checking the scalar target as this is not built within the dynamic
+// dispatched sources.
+#if (HWY_COMPILER_MSVC && !HWY_IS_DEBUG_BUILD) ||                   \
```

Comment:
why not just use `VQSORT_ENABLED` instead duplicating the macro?

Reply:
This is included by `quicksort.cpp`, which isn't dynamically dispatched, meaning the `VQSORT_ENABLED` flag would be set incorrectly due to having `HWY_TARGET == HWY_SCALAR`.